modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/selector/provider/ContentDashboardItemTypeItemSelectorProvider.java

```diff
@@ -96,6 +98,9 @@
 	private BooleanClause[] _getBooleanClauses(String keywords, Locale locale) {
 		BooleanQueryImpl booleanQueryImpl = new BooleanQueryImpl();
 
```

Comment:
We should contribute to the query from the different implementations. We were assuming too much. Each entity will have its own way of being searched.